scripts/linters/js_ts_linter.py

```diff
@@ -901,6 +901,91 @@ def _check_constants_declaration(self):
 
         return summary_messages
 
+    def _check_comments(self):
+        """This function ensures that comments follow correct style. Below are
+        some formats of correct comment style:
+        1. A comment can end with the following symbols: ('.', '?', ';', ',',
+        '{', '^', ')', '}', '>'). Example: // Is this is comment?
+        2. If a line contain any of the following words or phrases('@ts-ignore',
+        '--params', 'eslint-disable', 'eslint-enable', 'http://', 'https://')
+        in the comment.
+        """
+        if self.verbose_mode_enabled:
+            python_utils.PRINT(
+                'Starting comment checks\n'
+                '----------------------------------------')
+        summary_messages = []
+        files_to_check = self.all_filepaths
+        allowed_terminating_punctuations = [
+            '.', '?', ';', ',', '{', '^', ')', '}', '>']
+
+        # We need to ignore phrases below because if we use punctuation with
+        # these words it will mess up the whole meaning of the comment.
+        # Example: --params.login.user 'Joe'
+        # We can't use a period at the end of comment for such comment
+        # else it will mess up the whole meaning.
+        excluded_phrases = [
+            '@ts-ignore', '--params', 'eslint-disable', 'eslint-enable',
+            'http://', 'https://']
+
+        failed = False
+        with linter_utils.redirect_stdout(sys.stdout):
+            for filepath in files_to_check:
+                file_content = FILE_CACHE.readlines(filepath)
+                file_length = len(file_content)
+                for line_num in python_utils.RANGE(file_length):
+                    line = file_content[line_num].strip()
+                    next_line = ''
+                    previous_line = ''
+                    if line_num + 1 < file_length:
+                        next_line = file_content[line_num + 1].strip()
+                    if line_num + 1 < file_length:
+                        next_line = file_content[line_num + 1].strip()
+
+                    # Check if comment contains any excluded phrase.
+                    word_is_present_in_excluded_phrases = any(
```

Comment:
This variable name seems wrong -- shouldn't it be comment_includes_excluded_phrase (or similar)? (Though please note discussion above.)

Reply:
Done.